src/rgw/rgw_kafka.cc

```diff
@@ -599,12 +599,14 @@ class Manager {
                boost::optional<const std::string&> ca_location,
                boost::optional<const std::string&> mechanism,
                boost::optional<const std::string&> topic_user_name,
-               boost::optional<const std::string&> topic_password) {
+               boost::optional<const std::string&> topic_password,
+               boost::optional<const std::string&> brokers) {
```

Comment:
the new `brokers` parameter is not used in this function

Reply:
I overrided broker by using value from this paramter, i updated my code and move this code to new place, i think it is easier to read